File: krojac-ui/src/app/model/pantsRequest.ts
export class PantsRequest {
  waist!: number;        // waist circumference in cm
  hips!: number;         // hip circumference in cm
  length!: number;       // leg length in cm
  fit!: string;          // STANDARD, SLIM, LOOSE

  constructor(init?: Partial<PantsRequest>) {
    Object.assign(this, init);
  }
}

File: sbnz-integracija-projekta/service/src/main/java/com/ftn/sbnz/service/dto/PantsRequest.java
package com.ftn.sbnz.service.dto;

public class PantsRequest {
    private int waist;
    private int hips;
    private int length;
    private String fit; // STANDARD, SLIM, LOOSE

    public int getWaist() {
        return waist;
    }

    public void setWaist(int waist) {
        this.waist = waist;
    }

    public int getHips() {
        return hips;
    }

    public void setHips(int hips) {
        this.hips = hips;
    }

    public int getLength() {
        return length;
    }

    public void setLength(int length) {
        this.length = length;
    }

    public String getFit() {
        return fit;
    }

    public void setFit(String fit) {
        this.fit = fit;
    }
}
